### PR TITLE
fix: implement page object for vertical-navigation integration tests

### LINKED ISSUE #176: Fix IntegrationTest in vertical navigation

### PR DESCRIPTION
**Pull request by AlejandroYanes**
fixes #176

### CODE REVIEW

File: integration/specs/verticalNavigation-11.spec.js [hunk 1]
```diff
@@ -14,51 +11,75 @@ describe('VerticalNavigation expandable example', () => {
     });
 
     it('should open the "FOLDERS" menu when click on the "FOLDERS" button', () => {
-        browser.click(FOLDERS_BTN);
-        expect(browser.isVisible(ITEMS_CONTAINER)).toBe(true);
+        const verticalNavigation = new PageVerticalNavigation(VERTICAL_NAV);
+        const verticalSectionOverflow = verticalNavigation.getSctionOverflow(0);
```
Comment (LeandroTorresSicilia) [inline integration/specs/verticalNavigation-11.spec.js]: getSctionOverflow => getSectionOverflow

File: src/components/VerticalNavigation/pageObject/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     * @param {string} rootElement - The selector of the VerticalNavigation root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Returns a new VerticalItem page object of the element in item position.
+     * @method
+     *  @param {number} itemPosition - The base 0 index of the vertical item.
```
Comment (LeandroTorresSicilia) [inline src/components/VerticalNavigation/pageObject/index.js]: indentation

File: src/components/VerticalNavigation/pageObject/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        }
+        return null;
+    }
+
+    /**
+     * Returns a new VerticalSectionOverflow page object of the element in item position.
+     * @method
+     *  @param {number} itemPosition - The base 0 index of the vertical section overflow.
+     */
+    getSctionOverflow(itemPosition) {
```
Comment (LeandroTorresSicilia) [inline src/components/VerticalNavigation/pageObject/index.js]: getSectionOverflow

File: src/components/VerticalSectionOverflow/pageObject/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    click() {
+        $(this.rootElement).$('.rainbow-vertical-section-overflow_button').click();
+    }
+
+    /**
+     * Returns true when the overflow section is visible, false otherwise.
+     * @method
+     * @returns {bool}
+     */
+    isVerticalOverflowVisible() {
```
Comment (LeandroTorresSicilia) [inline src/components/VerticalSectionOverflow/pageObject/index.js]: only isVisible

Comment (reiniergs) [reply src/components/VerticalSectionOverflow/pageObject/index.js]: isVisible or isOpen?? or isExpended or is Collapsed

**Pull request merged**